examples/jsm/tsl/display/hashBlur.js

```diff
@@ -16,11 +16,11 @@ import { float, Fn, vec2, uv, sin, rand, degrees, cos, Loop, vec4, premultiplyAl
  */
 export const hashBlur = /*#__PURE__*/ Fn( ( [ textureNode, bluramount = float( 0.1 ), options = {} ] ) => {
 
-	const {
-		repeats = float( 45 ),
-		mask = null,
-		premultipliedAlpha = false
-	} = options;
+	textureNode = convertToTexture( textureNode );
+	bluramount = nodeObject( bluramount );
+	const repeats = nodeObject( options.size ) || float( 45 );
+	const mask = options.mask || null;
```

Comment:
Apparently, `mask` also needs to be compatible with `.sample()` interface, I think we can use `convertToTexture()` too.

Reply:
> Apparently, mask also needs to be compatible with .sample() interface, I think we can use convertToTexture() too.

Like so?
```js
const mask = ( options.mask !== null ) ? convertToTexture( options.mask ) : null;
```